src/GiftedChat.tsx

```diff
@@ -602,6 +616,13 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
   }
 
   onKeyboardWillHide = (_e: any) => {
+    // Use condition since the `onKeyboardWillHide` may be called
```

Comment:
Can you please also extract this code into a function of that class with an explicit name (and remove comments)?
Thanks